fars/booking/models.py

```diff
@@ -39,6 +41,10 @@ def get_metadata_form(self, data=None):
         else:
             return None
 
+    # It would be better if this was non-blocking
+    def notify_external_services(self):
```

Comment:
The naming of this method is a bit fuzzy, I did not realize initially that the notification related to deletion.
Perhaps a comment of that would suffice?

Reply:
It's not just related to deletion, it's supposed to happen whenever there is a change in data (i.e. a booking or an unbooking). The point is to let external services know so that they can fetch the updated data.

Reply:
Oooh, well spotted